website/www/site/i18n/home/calendar/en.yaml

```diff
@@ -0,0 +1,44 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+# TODO:
+# Placeholder texts should be updated later
```

Comment:
We need to update the placeholders for this calendar. Or should they be generated?

Reply:
there is implemented google calendar. It's in https://github.com/apache/beam/pull/13596